profile/profile.go

```diff
@@ -107,6 +108,8 @@ func StartContinuousProfile(profileDir string) {
 			// keeps the size of the log small while still providing lots of
 			// information.
 			StopCPUProfile()
+			runtime.GC()
+			debug.FreeOSMemory()
```

Comment:
I'm not sure this is desirable. I don't think the standard profiling tools run GC before collecting stats.